sockjs/eventsource_test.go

```diff
@@ -13,14 +13,20 @@ func TestHandler_EventSource(t *testing.T) {
 	h := newTestHandler()
 	h.options.ResponseLimit = 1024
 	go func() {
-		time.Sleep(1 * time.Millisecond)
-		h.sessionsMux.Lock()
-		defer h.sessionsMux.Unlock()
-		sess := h.sessions["session"]
-		sess.Lock()
-		defer sess.Unlock()
-		recv := sess.recv
-		recv.close()
+		var sess *session
+		for exists := false; !exists; {
```

Comment:
Maybe insert a millisecond Sleep here and to next loop to prevent CPU spikes during a test?